aten/src/ATen/core/function_schema.h

```diff
@@ -146,12 +154,17 @@ struct FunctionSchema {
     return is_varret_;
   }
   bool is_mutable() const {
-    return std::any_of(
-        arguments_.cbegin(), arguments_.cend(), [](const Argument& arg) {
-          const auto& aliasInfo = arg.alias_info();
-          return aliasInfo && aliasInfo.value().isWrite();
-        });
+    return is_inferred_from_custom_op_ ||
```

Comment:
So to be clear we are saying that it is mutable even if it doesn't have any inputs, add comment maybe ?